bundles/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/internal/device/DSMRDeviceRunnable.java

```diff
@@ -88,7 +89,7 @@ public void run() {
         } catch (final RuntimeException e) {
             logger.warn("DSMRDeviceRunnable stopped with a RuntimeException", e);
             portEventListener.onError(DSMRErrorStatus.SERIAL_DATA_READ_ERROR,
-                    Optional.ofNullable(e.getMessage()).orElse(""));
+                    Objects.requireNonNull(Optional.ofNullable(e.getMessage()).orElse("")));
```

Comment:
This looks nicer and its shorter for such constructs:

```suggestion
                    Objects.requireNonNullElse(e.getMessage(), "");
```

Reply:
Thanks, fixed (alsop on other places)